### DISCUSSION TITLE
.dwt problems in atom

### DISCUSSION THREAD

**Discussion by TonyHarmsworth** [atom/atom#22609] [Q&A]
With my previous editor, i could change the dwt and it would migrate to all of the files which used it. I could then upload those files and the indexes etc. would match.

In Atom, i can change the dwt, but it doesn't migrate the changes to the html files using it.

Any ideas?

**Comment by DeeDeeG** [comment]
After some quick research, it sounds like you are talking about Dynamic Web Templates for Microsoft Expression Web.

https://docs.microsoft.com/en-us/troubleshoot/iis/expression-web-faq

I'm not that familiar with how that is supposed to work...

Here's a tutorial I found... https://any-expression.com/expression-web/tutorials/dynamic-web-templates.htm

All I can think of is, are you are you are saving the files (Ctrl + S, or Cmd + S on a mac) when you're done editing? Is this linking something that needs to be done in the Microsoft Express Web software itself? What other editor were you working with that worked with this type of template? It might help those of us who are not familiar with DWTs to answer your question. Thanks.

**Reply by TonyHarmsworth** [reply]
Yes, it appears so. It was a brilliant system. If I added a new page, all i had to do was attach the dwt file in the header information and all the indexes surrounding the page automatically appeared. 

From what you say and others elsewhere, it was only an EW system so, basically, I'm stuffed.

It would be fine if I could buy Dreamweaver as that allowed some aspects to be kept, but Adobe no longer sell it. They are forcing me to spend a fortune on a 'membership' that I would only use maybe once a year to update the sit which I maintain on a volunteer basis. 

Thanks Microsoft for letting me down comprehensibly and without giving a damn!

In Atom, to add one new index item to the site, I have to change over fifty pages manually.

Thanks for your answer DeeDeeG

**Reply by DeeDeeG** [reply]
I'm seeing online that "Visual Web Developer" and "Visual Studio" have some ability to develop ASP.net websites. I'm not sure how compatible they are, but perhaps they can read and work with the old .dwt format?

https://forums.asp.net/t/1538468.aspx?Visual+Web+Developer+Express+vs+Visual+Studio+vs+Expression+Web

https://stackoverflow.com/questions/5694845/does-anyone-use-expression-web-and-visual-studio-together

alternativeto.net says Aptana Studio is a good alternative to DreamWeaver, but I have no idea if it supports .DWT files.

Best of luck.

**Reply by TonyHarmsworth** [reply]
Thanks for that. I'll take a look.
